app/views/layouts/mailer.html.slim

```diff
@@ -5,9 +5,14 @@ html
     = stylesheet_link_tag('mailer', media: 'all')
   body
     .main
-      .header style="background-image: url(#{attachments["header-pattern.png"]&.url})"
-        .brand
-          = image_tag(attachments["brave-payments.png"]&.url, alt: t("shared.app_title"), height: 31, width: 268)
+      - if @_controller.is_a?(PromoMailer)
```

Comment:
this is kind of hacky

i think we should have a separate PromoMailer layout

or (i'm not sure how to do this): a layout which inherits from `mailer.html.slim`

Reply:
i agree, and will look into this but at a lower priority for now

Reply:
Now using a PromoMailer layout